WrightTools/artists/_helpers.py

```diff
@@ -543,7 +548,13 @@ def full(arr):
             ll = orig[idx[0] + 0, idx[1] + 0]
             lr = orig[idx[0] + 0, idx[1] + 1]
             out[idx] = np.mean([ul, ur, ll, lr])
-    return X, Y
+    if zi is not None:
+        warnings.warn(
+            "zi argument is not used in pcolor_helper and is not required", DeprecationWarning
```

Comment:
Can we use wt_exceptions.VisibleDeprecationWarning instead? By default DeprecationWarnings are silent